### DISCUSSION TITLE
为什么我的页面不能滚动，而且显示了滚动条。

### DISCUSSION THREAD

**Discussion by trappedlamb** [xfgryujk/blivechat#233] [通用]
。。。。

**Comment by SSSSSSSan** [comment]
只有直播姬有这个问题 把浏览器源的自定义css全删了

**Reply by trappedlamb** [reply]
不是直播鸡的问题，我在浏览器打开房间也是有滚动条，用作者的公共服务就没有。

**Comment by trappedlamb** [comment]
有大佬知道是怎么回事儿吗？我套了CF和NGINX。只能用公共服务了吗。。。

**Reply by SSSSSSSan** [reply]
我没法复现..

**Reply by trappedlamb** [reply]
用公共服务器就能滚动到底部呀，我自己搭建的就不行。是CJ没运行吗还是其他问题。我也不太懂呢。就是页面不能滚动到底部，导致一直在顶部。

**Reply by trappedlamb** [reply]
你们搭建的网页是和开发者的页面一样吗，我的网页就不一样，主要问题就是房间页面不能滚动到底部！折腾了两天，把这些注意事项都看了个遍。。。但是可以不用，但我就想知道是怎么回事儿，如鲠在喉。

**Reply by SSSSSSSan** [reply]
> 用公共服务器就能滚动到底部呀，我自己搭建的就不行。是CJ没运行吗还是其他问题。我也不太懂呢。就是页面不能滚动到底部，导致一直在顶部。

我这里完全没问题所以不知道发生了什么

**Reply by trappedlamb** [reply]
我可以把我那个房间的链接贴上来你看看吗？如果你乐意的话

**Comment by trappedlamb** [comment]
https://mg.lamb.free.hr/room/test?roomKeyType=2&blockGiftDanmaku=false&showDebugMessages=true&importPresetCss=true&lang=zh
 这是我的测试房间地址，哪位大佬能看看到底出了什么问题吗？滚动条一直在上面

**Reply by SSSSSSSan** [reply]
你css呢....

**Reply by trappedlamb** [reply]
对呀，我也想问呀。我在/custom_public 里面预设了。但是html好像没有读取吗？开发者没有讲怎么操作呀。我都是依葫芦画瓢的。。。

**Comment by SSSSSSSan** [comment]
> https://mg.lamb.free.hr/room/test?roomKeyType=2&blockGiftDanmaku=false&showDebugMessages=true&importPresetCss=true&lang=zh
>  这是我的测试房间地址，哪位大佬能看看到底出了什么问题吗？滚动条一直在上面

你css呢...

**Comment by trappedlamb** [comment]
开发者大佬能看到吗～我按着你的步骤，然后就出现这个问题，没有css？

**Reply by xfgryujk** [reply]
It works on my machine
